log/logger.go

```diff
@@ -262,9 +257,25 @@ func Critf(format string, rest ...interface{}) {
 	logPrintf(Critical, format, rest...)
 }
 
-// Fatalf logs if Warning level is on.
+// Fatalf logs if Warning level is on and panics or exits.
 func Fatalf(format string, rest ...interface{}) {
 	logPrintf(Fatal, format, rest...)
+	if *fatalPanics {
+		panic("aborting...")
+	}
+	os.Exit(1)
```

Comment:
it actually is covered by the test but spawning a shell doesn't trigger coverage properly